Resources/data/pumoodle/repository/pmksearch/version.php

```diff
@@ -26,6 +26,6 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2016062100;        // The current plugin version (Date: YYYYMMDDXX)
+$plugin->version   = 2016071200;        // The current plugin version (Date: YYYYMMDDXX)
```

Comment:
@rubenrua Yes, it did.
https://github.com/teltek/PuMuKIT2-moodle-bundle/pull/15/files#diff-17daad81479092c048468835b8f3ccd2